test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -297,6 +297,41 @@ test.describe( 'Cover', () => {
 
 		expect( isClickable ).toBe( false );
 	} );
+
+	test( 'can use focal point picker to set the focal point of the cover image', async ( {
+		editor,
+		coverBlockUtils,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+		const coverBlock = editor.canvas.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+
+		await coverBlockUtils.upload(
+			coverBlock.getByTestId( 'form-file-upload-input' )
+		);
+
+		await editor.page.keyboard.press( 'ArrowUp' );
+
+		const focalPointLeft = page.getByLabel( 'Focal point left position' );
+
+		const focalPointTop = page.getByLabel( 'Focal point top position' );
```

Comment:
```suggestion
		const focalPointLeft = page.getByRole( 'spinbutton', {
			name: 'Focal point left position',
		} );
		const focalPointTop = page.getByRole( 'spinbutton', {
			name: 'Focal point top position',
		} );
```

Use accessible selectors as much as possible. See https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/e2e/README.md#use-accessible-selectors